File: examples/Java/SortArray.java
import java.util.Random;

public class SortArray { 
  public static int[] fillArr() {
    int[] arr = new int[10];
    Random rand = new Random();
    for (int i = 0; i < 10; i++) {
      arr[i] = rand.nextInt(100) + 1;
    }
    return arr;
  }

  public static int[] sortArr(int[] arr) {
    int swaps = 0;
    for (int i = 0; i < arr.length; i++) {
      for (int j = 0; j < arr.length - 1; j++) {
        if (arr[j] > arr[j + 1]) {
          int temp = arr[j];
          arr[j] = arr[j + 1];
          arr[j + 1] = temp;
          swaps ++;
        }
      }
    }
    return arr;
  }

  public static void main(String[] args) {
    while (true) {
      int[] arr = fillArr();
      System.out.println("Original array:");
      for (int i = 0; i < arr.length; i++) {
        System.out.print(arr[i] + " ");
      }
      System.out.println();
      int[] sortedArr = sortArr(arr);
      System.out.println("Sorted array:");
      for (int i = 0; i < sortedArr.length; i++) {
        System.out.print(sortedArr[i] + " ");
      }
      System.out.println();
      try {
        Thread.sleep(1000);
      } catch (InterruptedException e) {
      }
      
    }
  }
}

File: examples/Python/sortArray.py
import random
import time

def fillArray():
    arr = []
    for i in range(10):
        arr.append(random.randint(1, 100))
    return arr

def sortArray(arr):
    swaps = 0
    for i in range(len(arr)):
        for j in range(len(arr)-1):
            if arr[j] > arr[j+1]:
                arr[j], arr[j+1] = arr[j+1], arr[j]
                swaps = swaps + 1
    return arr

def main():
    try:

        import lightrun # this would generally be together with other imports at the top of the file
        lightrun.enable(
            company_key='xxxxx-xxxxx-xxxxx-xxxxx-xxxxx') 
    except ImportError as e:
        print("Error importing Lightrun: ", e)

    while True:
        arr = fillArray()
        print("Original array:", arr)
        arr = sortArray(arr)
        print("Sorted array:", arr)
        time.sleep(1)

if __name__ == "__main__":
    main()
